### PR TITLE
Convert 2 ports to 1 port on the CallAutomation Livetranscription

### PR DESCRIPTION
**Pull request by BalajiUmmadisetti**
Convert 2 ports to 1 port on the CallAutomation Livetranscription

Tested:
connection ID: 0f004480-228e-4727-9d25-b7463bacb305
'correlationId': '0f8504ff-b328-4a50-9f70-3b4e057aa351'

## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: callautomation-live-transcription/main.py [hunk 1]
```diff
@@ -1,15 +1,14 @@
 
 import ast
```
Comment (v-vdharmaraj) [inline callautomation-live-transcription/main.py:2]: Please test the application and share the correlation id's along with the description

Comment (BalajiUmmadisetti) [reply callautomation-live-transcription/main.py:2]: connection ID: 0f004480-228e-4727-9d25-b7463bacb305
'correlationId': '0f8504ff-b328-4a50-9f70-3b4e057aa351'

File: callautomation-live-transcription/main.py [hunk 2]
```diff
@@ -81,74 +84,88 @@
...
-        end_silence_timeout=15, 
-        dtmf_inter_tone_timeout=5,
-        play_prompt=play_source,
-        operation_context=context)
-    app.logger.info("handle_recognize : data=%s",recognize_result) 
-
-def handle_play(call_connection_id, text_to_play, context):     
+app = Quart(__name__)
+
+
```
Comment (v-vdharmaraj) [inline callautomation-live-transcription/main.py:89]: remove extra spaces

File: callautomation-live-transcription/readme.md [hunk 3]
```diff
@@ -28,30 +27,20 @@ This sample application is also capable of making multiple concurrent inbound ca
 
 Create and activate python virtual environment and install required packages using following command 
 ```
+python -m venv venv
+venv\Scripts\activate
```
Comment (v-vdharmaraj) [inline callautomation-live-transcription/readme.md:31]: why do we needs to add this?

**Review by v-vdharmaraj — APPROVED** [review]

**Pull request merged**